ViAn/GUI/mainwindow.cpp

```diff
@@ -981,10 +984,11 @@ void MainWindow::add_analysis_to_queue(ANALYSIS_TYPE type, QString name, MyQTree
  * filehandler workspace accordingly.
  */
 void MainWindow::on_action_choose_workspace_triggered() {
-    QString dir = QFileDialog::getExistingDirectory(this, tr("Choose Workspace"),this->file_handler->get_work_space().absolutePath().toStdString().c_str());
+    const std::string workspace = this->file_handler->get_work_space().absolutePath().toStdString();
+    QString dir = QFileDialog::getExistingDirectory(this, tr("Choose Workspace"), workspace.c_str());
     if (!dir.isEmpty()) {
         this->file_handler->set_work_space(dir.toStdString() + "/");
-        set_status_bar("New wokspace set to " + this->file_handler->work_space);
+        set_status_bar("New wokspace set to " + workspace);
```

Comment:
Is worksapace one word? (I think yes) If so then in the line above if should be one word as well in the function "set_work_space()"